labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise02.scala

```diff
@@ -39,6 +39,8 @@ object ListManipulationExercise02 {
    * in a one-liner.
    */
   def separateTheYoungFromTheOld(persons: List[Person]): List[List[String]] = {
+    
+    
```

Comment:
Huh? Where did this go? I guess I'll have to update this at some point, because my solution here disappeared. I had done the unnecessary one-liner

```
(persons groupBy (_.age < 18)).values.toList sortBy (_.head.age) map (_ sortBy (_.age) map (_.firstName))
```